### PR TITLE
update v0.1.2

### PR DESCRIPTION
**Pull request by Arishamays1**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -10,15 +10,15 @@ source:
...
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
     - hatchling
+    - wheel
   run:
     - python >=3.6
```
Comment (ELundby45) [inline recipe/meta.yaml]: The pinning on `python` should be removed:
`- python >=3.6` -> `- python`

File: recipe/meta.yaml [hunk 2]
```diff
@@ -10,15 +10,15 @@ source:
   sha256: 3e2f5826578e683999b93716285b3b1f344f157bf75fa9ce0a797564e742f062
 
 build:
-  noarch: python
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:13]: Skip py<36, see https://github.com/ipython/comm/blob/f4190da20bacd94d669819c725a7d0881dfe9c65/pyproject.toml#L11

Comment (Arishamays1) [reply recipe/meta.yaml:13]: Thanks made an amendment

Comment (ELundby45) [reply recipe/meta.yaml:13]: @Arishamays1 Could you make one more minor change the `<` is missing.

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Thanks Eric! Made the change!

**Review by varlackc — COMMENTED** [review]
since we do not have this package version in defaults we might want to reset the build number to zero

https://github.com/AnacondaRecipes/comm-feedstock/blob/83526e6c0946840f73e0b2bd437748d36821b924/recipe/meta.yaml#L14


This might be a recommendation and is up to debate, but I think that the following url might be a bit better for the `doc_url` for now:

doc_url: https://github.com/ipython/comm/blob/main/README.md

**Comment by Arishamays1** [comment]
> since we do not have this package version in defaults we might want to reset the build number to zero
> 
> https://github.com/AnacondaRecipes/comm-feedstock/blob/83526e6c0946840f73e0b2bd437748d36821b924/recipe/meta.yaml#L14
> 
> This might be a recommendation and is up to debate, but I think that the following url might be a bit better for the `doc_url` for now:
> 
> doc_url: https://github.com/ipython/comm/blob/main/README.md

Thanks Maxwell. My last recent commit changed the build number back to zero. I can change the doc_url!

**Review by varlackc — APPROVED** [review]
LGTM

**Review by ELundby45 — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**